### PR TITLE
Code reorganization

### PR DESCRIPTION
**Pull request by vadimkozak**
I made small code reorganization: 
 - changed access levels, so after that you can create pod/carthage library 
 - added skeleton of public methods documentation
 - better demo 
 - some minor changes

### CODE REVIEW

**Comment by vadimkozak** [comment]
Also better demo added

**Comment by FZambia** [comment]
@vadimkozak hi, thank you, great to see interest to this library!

Just start looking at changes, you are using this code in demo:

```
guard let status = self.client?.status else {return}
if status == .new || status == .disconnected {
```

At this place you rely on non public `status` property. I'd like to keep this property private for now, so maybe it's better to rely on `connect` and `disconnect` events at moment to manage app state.

Also could you tell me what `//MARK -` mean?

**Comment by vadimkozak** [comment]
hi, 
Agree with you about access to ‘status’ property. But it was internal before.
//MARK - 
is used for bettercode navigation and separate different parts of code to different sections.

**Comment by FZambia** [comment]
> Agree with you about access to ‘status’ property. But it was internal before.

I mean that you should not use it [in demo](https://github.com/centrifugal/centrifuge-swift/pull/3/files#diff-51095b79ef7811c29616112b3309d565R43) because library users won't have access to it.

**Comment by vadimkozak** [comment]
@FZambia thanks, did it here 545d474

**Comment by FZambia** [comment]
@vadimkozak thanks, one more thing is that `getSubscriptionToken` should remain private - it's not used by library users.

**Comment by FZambia** [comment]
@vadimkozak thanks! But `refreshWithToken` also internal func :) Token refresh mechanism fully based on delegate method of `CentrifugeClientDelegate` - `onRefresh`

**Comment by FZambia** [comment]
`rpc` and `send` methods should remain public, `history` method should remain private. the same for `unsubscribe`.

Also wrong comment near `disconnect` method.

File: Demo/AppDelegate.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```
Comment (FZambia) [inline Demo/AppDelegate.swift]: This does not work in XCode 9.4 - see 34f7beb4c4d215158bbf67eb4d1e5b78cc25e336

Comment (vadimkozak) [reply Demo/AppDelegate.swift]: yes, it was changed to swift 4.2 and Xcode 10.1

Comment (FZambia) [reply Demo/AppDelegate.swift]: Does the existing code not working in 10.1?

Comment (vadimkozak) [reply Demo/AppDelegate.swift]: yes, its doesn't work with new Xcode

**Comment by vadimkozak** [comment]
Understood, thanks for clarification!

>Is this a convenient way for Swift libs to declare public API?

There are no some docs or something about that someone use it to describe public functions like in Java, someone not.

**Comment by FZambia** [comment]
> Someone use it to describe public functions like in Java, someone not.

Will try to think about this but for now I suppose we can skip this step.

Thanks a lot for your improvements, will merge changes and then I need to resolve some conflicts with my other branch with several important fixes (#4 ).

**Comment by vadimkozak** [comment]
Thank you for your project, its simply great!

**Pull request merged**